pkg/resourcemanager/config/env.go

```diff
@@ -157,3 +166,13 @@ func ParseStringListFromEnvironment(variable string) []string {
 	}
 	return items
 }
+
+func ParseOperatorMode(value string) (OperatorMode, error) {
```

Comment:
minor: Maybe throw a comment on this?

Reply:
Oh, the updated version does have a comment! :)